drivers/windows/windows.go

```diff
@@ -148,6 +149,15 @@ func (d *driver) parseNetworkOptions(id string, genericOptions map[string]string
 			config.DNSSuffix = value
 		case DNSServers:
 			config.DNSServers = value
+		case MacPool:
+			config.MacPools = make([]hcsshim.MacPool, 0)
+			s := strings.Split(value, ",")
+			for i := 0; i < len(s)-1; i += 2 {
```

Comment:
The mac pool range must be specified in pair. For odd number length I want to ignore the last value. If you want we can throw an error in case of odd numbers.